src/Catalyst.Simulator/Helpers/PasswordRegistryHelper.cs

```diff
@@ -29,13 +29,12 @@ namespace Catalyst.Simulator.Helpers
 {
     public static class PasswordRegistryHelper
     {
-        public static void AddPassword(PasswordRegistry passwordRegistry, PasswordRegistryTypes passwordRegistryTypes, string password)
+        public static void AddPassword(PasswordRegistry passwordRegistry,
+            PasswordRegistryTypes passwordRegistryTypes,
+            string password)
         {
             var secureString = new SecureString();
-            foreach (var character in password)
-            {
-                secureString.AppendChar(character);
-            }
+            foreach (var character in password) secureString.AppendChar(character);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)